.github/workflows/ci.yaml

```diff
@@ -0,0 +1,41 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
```

Comment:
shrug

Reply:
Well I'm **not** going to rename to `main`.